src/createAction.js

```diff
@@ -29,7 +29,7 @@ export default function createAction(type, payloadCreator, metaCreator) {
     return action;
   };
 
-  actionHandler.toString = () => type;
+  actionHandler.toString = () => type.toString();
```

Comment:
Yes, there's been a [test](https://github.com/acdlite/redux-actions/blob/master/src/__tests__/handleActions-test.js#L26) for this for quite a while (added in https://github.com/acdlite/redux-actions/commit/dd8ef00dd8883f8159d4de43f516ea52514b284e), and I didn't want to cause a "regression" with this change. I'm guessing it was fixed at the time because it easy and quick to fix.

Here's the original issue where the BDFL recommended against using it: https://github.com/acdlite/redux-actions/issues/20.